pkg/controllers/management/nodedriver/dynamic_driver.go

```diff
@@ -236,10 +254,20 @@ func (d *Driver) copyBinary(cacheFile, input string) (string, error) {
 		if err != nil {
 			return "", err
 		}
-		driverName = strings.Split(path.Base(u.Path), "_")[0]
-		if !strings.HasPrefix(driverName, "docker-machine-driver-") {
+
+		if !strings.HasPrefix(path.Base(u.Path), "docker-machine-driver-") {
```

Comment:
lets make a constant for docker-machine-driver as it being referenced multiple times